Autopilot/Path Manager/ProbeDrop.h

```diff
@@ -0,0 +1,13 @@
+/* 
+ * File:   ProbeDrop.h
+ * Author: banky
```

Comment:
Your IDE probably has a location to customize this, so you don't have to change it every time you create a new file.